azure-pipelines.yml

```diff
@@ -16,9 +16,9 @@ steps:
     version: 2.2.x
```

Comment:
Do you need .NET Core 2 and .NET Core 3?

Reply:
It was needed before, now only 3.x it's fixed in separate PR, thanks